integration/commands_administration_test.go

```diff
@@ -574,6 +574,26 @@ func TestCommandsAdministrationBuildInfo(t *testing.T) {
 	assert.True(t, ok)
 }
 
+func TestCommandsAdministrationBuildInfoFerretdbExtensions(t *testing.T) {
+	setup.SkipForMongoDB(t, "FerretDB-specific command's extensions")
+
+	t.Parallel()
+	ctx, collection := setup.Setup(t)
+
+	var actual bson.D
+	command := bson.D{{"buildInfo", int32(1)}}
+	err := collection.Database().RunCommand(ctx, command).Decode(&actual)
+	require.NoError(t, err)
+
+	doc := ConvertDocument(t, actual)
+
+	ferretdbFeatures, ok := must.NotFail(doc.Get("ferretdbFeatures")).(*types.Document)
```

Comment:
We should not use must.NotFail in this case :smile:. Because it will panic instead of test failure if `ferretdbFeatures` is not set by the implementation. I think you saw that with `test-integration-mongodb`.

Something like below will report test has failed.
```
v, err := doc.Get("ferretdbFeatures")
require.NoError(t, err)
```